### ISSUE TITLE
[Screen reader] `if` block navigation with inline inputs and shadow blocks

### ISSUE THREAD

**Issue by microbit-robert** [rachel-fenichel/blockly-keyboard-experimentation#767]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Inline inputs
In the proposed keyboard navigation changes implemented in the screen reader branch for `if` blocks, the up / down arrows visit the value inputs in the `else if` clauses, but this does not work for `if` blocks that have inline inputs. This impacts the `if` block in MakeCode. If we don't set inline inputs, the block looks rather unpalatable (see both examples below). If there is a way of constructing this block without having to set inline inputs, please let us know, otherwise, it seems like some changes to the `getValidationFunction` on `LineCursor` are required. 

<img width="276" height="310" alt="Image" src="https://github.com/user-attachments/assets/afd8ee71-3b99-4f21-8243-d7fd600d2d4a" />
<img width="221" height="450" alt="Image" src="https://github.com/user-attachments/assets/b92255e0-a2a0-4f50-9654-bfee791be6b6" />

### Shadow blocks
In the MakeCode case, the boolean logic value inputs are initially shadow blocks. When using the left and right arrows to  navigate to these value inputs, the whole field row is not read out by the screen reader, which is now expected. I think this is because we go down [this path in the code](https://github.com/google/blockly/blob/add-screen-reader-support-experimental/core/block_svg.ts#L235-L238), therefore, the aria-label is just that of the shadow block true/false dropdown, and does not include any other text.

### Reproduction steps

1. Go to https://screen-reader-preview.review-pxt.pages.dev/#editor - this is a version of MakeCode that has the latest screen reader work integrated
2. Insert an if / else block from the logic category in toolbox
3. Use the '+' button to add an 'else if' case
4. Observe that the up / down arrow navigation doesn't visit the true / false value in the 'else if' row
5. Observe that when navigating to these true / false value inputs using the left / right arrow keys, only the aria attributes for this value input are announced, and the 'else if' text is ignored

### Stack trace

```shell

```

**Comment by BenHenning** [comment]
@gonfunko you might be able to dig into this more quickly than me when you're back if I can't get to this first (will update from 'todo' if I actually get to starting this). This feels partly an issue with navigation and partly and issue with how we're compiling the block labels.

**Comment by BenHenning** [comment]
Haven't made progress so de-assigning so someone else can possibly pick this up.

**Comment by microbit-robert** [comment]
We have a workaround in place already for user testing, so this is not a priority for January for us.

**Comment by maribethb** [comment]
I think this issue is covered by https://github.com/RaspberryPiFoundation/blockly/issues/9617 now
